index.html

```diff
@@ -15,16 +15,26 @@
         <div class="toolbar">
           <a href="">John Doe</a>
           <i><img src="./assets//menu.png" /></i>
+          <nav class="menu">
```

Comment:
Good job! Your navbar looks very similar to the one in Figma, but to make it the same thing, I'd recommend increasing the height of your navbar.

![image](https://user-images.githubusercontent.com/34779928/193093508-74409721-0b07-4cdf-a89d-ff38d75cbfa7.png)